solr/core/src/java/org/apache/solr/handler/designer/SchemaDesignerConfigSetHelper.java

```diff
@@ -525,40 +495,22 @@ void deleteStoredSampleDocs(String configSet) {
 
   @SuppressWarnings("unchecked")
   List<SolrInputDocument> getStoredSampleDocs(final String configSet) throws IOException {
-    List<SolrInputDocument> docs = null;
-
-    final URI uri;
-    try {
-      uri =
-          collectionApiEndpoint(BLOB_STORE_ID, "blob", configSet + "_sample")
-              .setParameter(CommonParams.WT, "filestream")
-              .build();
-    } catch (URISyntaxException e) {
-      throw new SolrException(SolrException.ErrorCode.BAD_REQUEST, e);
-    }
-
-    HttpGet httpGet = new HttpGet(uri);
+    var request = new GenericSolrRequest(SolrRequest.METHOD.GET, "/blob/" + configSet + "_sample");
+    request.setRequiresCollection(true);
+    request.setResponseParser(new InputStreamResponseParser("filestream"));
```

Comment:
I played with this method a bit, hoping to remove InputStream low level stuff but there seems to be something special about "filestream" so I left this aspect as is.  Besides, the response handling is pretty concise nonetheless.